Short-Answer/Algorithms_Answers.md

```diff
@@ -2,14 +2,29 @@
 
 ## Exercise I
 
-a)
+a) O(n) or linear because n is unknown. If we choose n to be 2 the function would run 8 times, however if we choose 10 it runs 1000 times.
```

Comment:
Correct runtime complexity but your explanation is a bit off. A linear runtime of O(n) means that our function will iterate `n` times or `n+constant` times.